spec/test.spec.js

```diff
@@ -323,76 +323,79 @@ describe('S3Adapter tests', () => {
 
     beforeEach(() => {
       options = {
+        presignedUrl: false,
         directAccess: true,
         bucketPrefix: 'foo/bar/',
         baseUrl: 'http://example.com/files',
       };
     });
 
     it('should get using the baseUrl', () => {
-      const s3 = new S3Adapter('accessKey', 'secretKey', 'myBucket', options);
+      const s3 = new S3Adapter('accessKey', 'secretKey', 'my-bucket', options);
       expect(s3.getFileLocation(testConfig, 'test.png')).toEqual('http://example.com/files/foo/bar/test.png');
     });
 
-    it('should get direct to baseUrl', () => {
```

Comment:
Why have these tests been removed? Could you re-add them and just add the new tests for the new feature?